bitrisescript/src/bitrisescript/bitrise.py

```diff
@@ -183,6 +184,31 @@ async def download_log(build_slug: str, artifacts_dir: str, poll_interval: int =
         log.error(f"Bitrise has no log for build '{build_slug}'. Please check https://app.bitrise.io/build/{build_slug}")
 
 
+async def dump_perfherder_data(artifacts_dir: str) -> None:
+    """Dumps any detected PERFHERDER_DATA log lines to stdout.
+
+    Perfherder is hardcoded to parse `public/live_backing.log` for lines that
+    start with PERFHERDER_DATA. Scan the bitrise log and forward any such lines
+    to stdout so Perfherder can find them.
+
+    Once bug 1646502 is fixed and all tasks relying on this function have been
+    migrated over to the new ingestion format, we can remove this.
+
+    Args:
+        artifacts_dir (str): Path to the artifact directory.
+    """
+    path = Path(artifacts_dir) / "bitrise.log"
+    if not path.is_file():
+        log.warning(f"Not scanning for Perfherder data, {path} does not exist!")
+        return
+
+    perfherder_data_lines = [line for line in path.read_text().splitlines() if line.startswith("PERFHERDER_DATA")]
+    print(perfherder_data_lines)
```

Comment:
Should we use `log.info` instead of `print`, or is this intentional?

Reply:
Uhh.. that is a leftover debug statement I forgot to remove ><